tools/Environments/DevHome.Environments/Strings/en-us/Resources.resw

```diff
@@ -111,15 +111,15 @@
     <comment>Text for terminate button in the context menu that will turn off the virtual machine. This button turns off the power to the virtual machine without gracefully shutting it down first.</comment>
   </data>
   <data name="SortByName.Content" xml:space="preserve">
-    <value>Name</value>
-    <comment>Text for 1st option for sort by drop down box</comment>
+    <value>Name: A-Z</value>

```

Comment:
I wonder if translation is going to not like this. Isn't Ascending/Descending more commonly used?

Reply:
This is what the designers wanted. I agree, ascending/descending would be better. Since we want parity with the Setup pages, @bbonaby if you don't have any reservations, let's both change this.